analytics/lib/counts.py

```diff
@@ -114,7 +135,12 @@ def do_aggregate_to_summary_table(stat, end_time, interval):
                'property' : stat.property,
                'interval' : interval}
 
+        start = time.time()
         cursor.execute(realmcount_query, {'end_time': end_time})
+        end = time.time()
+        rows_changed = cursor.rowcount
+        logging.info("RealmCount aggregation %2.1f ms/%sR"
```

Comment:
I think unlike something I said earlier let's also start the line with the stat.property; will be better for grepping through the logs